scripts/vm/hypervisor/vmware/discover_networks.py

```diff
@@ -0,0 +1,272 @@
+#!/usr/bin/env python3
```

Comment:
This network discovery script is for discovering networks from VMware datacenter. Admin can run this externally and use the output to create networks in CS for smoother import of existing unmanaged VMs. It will be installed in cloudstack-common package and hence not called anywhere in code.

Reply:
ok, maybe I missed a comment or a usage message?